static/_redirects

```diff
@@ -50,6 +50,7 @@
 /calico/latest/security/encrypt-cluster-pod-traffic /calico/latest/network-policy/encrypt-cluster-pod-traffic 301
 
 /calico-cloud/image-assurance/scan-image-registries /calico-cloud/image-assurance/scanners/cli-based-scanner 301
+/calico-cloud/image-assurance/get-started/connect/usage-metrics /calico-cloud/image-assurance/operations/usage-metrics 301
```

Comment:
I think this redirect is wrong as it was not under image-assurance before or after these changes?
```suggestion
/calico-cloud/get-started/connect/usage-metrics /calico-cloud/operations/usage-metrics 301
```

Reply:
I have a separate PR to fix IA redirects.